application/controllers/UserGroupController.php

```diff
@@ -158,8 +150,6 @@ public function actionViewGroup($ugid, bool $header = false)
             $aNewUserListData = CHtml::listData($aUsers, 'uid', function ($user) {
                 return \CHtml::encode($user->users_name) . " (" . \CHtml::encode($user->full_name) . ')';
             });
-            // Remove group owner because an owner is automatically member of a group
```

Comment:
The comment is being removed. Doesnt apply anymore?
If so, the unset below, should til be there?

Reply:
You're right … not related to this commit.

OIn 6X : we can remove the owner form user in group, not in 5X
Unsure we need to remove it